### ISSUE TITLE
Failed to upload project to Lambda from Eclipse

### ISSUE THREAD

**Issue by AlfonsoBonillasALIO2019** [aws/aws-toolkit-eclipse#233]
I am trying to upload project to lambda from Eclipse using the AWS upload tool..

I am getting this error:

![image](https://user-images.githubusercontent.com/47362953/120566281-c40a9900-c3c3-11eb-8753-8e010f41d8c4.png)

I did change the build settings to use an "older" java jdk (java 8) but still the same problem.

![image](https://user-images.githubusercontent.com/47362953/120566332-df75a400-c3c3-11eb-8abb-64dfdda6ea70.png)

Anybody out there knows what could be cause?

**Comment by SubZero5z** [comment]
Yeah i have the same issue.Were you able to resolve it @AlfonsoBonillasALIO2019  ?
